### DISCUSSION TITLE
Grid resolution for narrow fire

### DISCUSSION THREAD

**Discussion by Tbuttts21** [firemodels/fds#12142] [General]
Was hoping to get some input on deciding a grid resolution surrounding a narrow fire. I have a fire that is very long but only about a meter across. As a rule of thumb, I've been doing about 10 cells across the smallest dimension of the fire (i.e., 10 centimeter sized cells). However, this causes issues when the target of my fire is far away because my total cell count ends up being impracticably large.

My question is: is there any guidance on the cell size/grid resolution required across the surface of a fire? I know that it's important to have enough cells across the surface of the fire in order for buoyancy effects to take place. Is 10 cells too much? too few? Any help would be appreciated.

**Comment by rmcdermo** [comment]
As with many scenarios, it depends on what is being asked of the code.  If heat flux to a distant target is the main question, then you probably need to think most about flame height and resolving the angular path to the target.  FDS has been shown to get reasonable flame heights with 5 cells across the plume.  But you may need a lot of radiation angles to make sure you avoid the star pattern from the discrete ordinates solver, like 1000+.  This can be offset somewhat by reducing the temporal frequency of the radiation solution.  See ANGULAR_INCREMENT and TIME_STEP_INCREMENT on the RADI line.  You should also consider attenuation effects of H2O and CO2 in the atmosphere.  You may need to adjust PATH_LENGTH to properly account for this.

**Reply by Tbuttts21** [reply]
Yes, the main goal is to find the heat flux to a distant target. I've read in some papers that it's important to have enough cells across the surface of the fire in order for the periodic formation of toroidal vortexes in the fire. Are there any resources that point towards that 5 cell across the plume metric?

**Reply by rmcdermo** [reply]
Look at the Heskestad flame height in the FDS validation guide.  If the target is distant, the fire becomes more or less a point source, or if the geometry matters at all, more of cylindrical source.  Then what matters is that you have the right heat release and radiant fraction, both things you are specifying in the model.

**Reply by Tbuttts21** [reply]
Thank you, will take a look at this. I appreciate your help.
